src/adapters/ucfunnel.js

```diff
@@ -0,0 +1,95 @@
+import * as Adapter from './adapter.js';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import {ajax} from 'src/ajax';
+import {STATUS} from 'src/constants';
+
+const VER = 'ADGENT_PREBID-2017051801';
+const BIDDER_CODE = 'ucfunnel';
+
+var ucfunnelAdapter = function ucfunnelAdapter() {
+  function _callBids(params) {
+    var bids = params.bids || [];
+
+    bids.forEach((bid) => {
+      try {
+        ajax(buildOptimizedCall(bid), bidCallback, undefined, { withCredentials: true });
+      } catch (err) {
+        utils.logError('Error sending ucfunnel request for placement code ' + bid.placementCode, null, err);
+      }
+
+      function bidCallback(responseText) {
+        try {
+          utils.logMessage('XHR callback function called for placement code: ' + bid.placementCode);
+          handleRpCB(responseText, bid);
+        } catch (err) {
+          if (typeof err === 'string') {
+            utils.logWarn(`${err} when processing ucfunnel response for placement code ${bid.placementCode}`);
+          } else {
+            utils.logError('Error processing ucfunnel response for placement code ' + bid.placementCode, null, err);
+          }
+
+          // indicate that there is no bid for this placement
+          let badBid = bidfactory.createBid(STATUS.NO_BID, bid);
+          badBid.bidderCode = bid.bidder;
+          badBid.error = err;
+          bidmanager.addBidResponse(bid.placementCode, badBid);
+        }
+      }
+    });
+  }
+
+  function buildOptimizedCall(bid) {
+    bid.startTime = new Date().getTime();
+
+    var host = window.location.host,

```

Comment:
Since you have used `let` at other places i think here also you can replace `var` with `let`